src/containers/User/UserArea.tsx

```diff
@@ -428,10 +428,16 @@ const OrgSelector: React.FC<{ user: User; orgs: OrganisationSimple[]; onLogin: F
       ) as any,
       value: LOGIN_AS_NO_ORG,
     })
+
+  const logoIsLink = viewport.width < 860
```

Comment:
This is using the same arbitrary "860px" as yesterdays header tweaks. Ideally we'd put this value in a constant somewhere, but not sure how to make a constant that can be available to both the LESS file and React components.

Reply:
Not possible I believe.